vulcanai2/models/cnn.py

```diff
@@ -99,17 +81,17 @@ def _create_network(self, **kwargs):
             self._create_classification_layer(
                 self.conv_flat_dim, kwargs['pred_activation'])
             
-            if torch.cuda.is_available():
-                for module in self.modules():
-                    module.cuda()
+        if torch.cuda.is_available():
```

Comment:
Do you need to do this for each module? can you not just apply it to `self.cuda()`?